src/components/cartModal/CartModal.tsx

```
@@ -0,0 +1,39 @@
+
+  const cartItems = getData<CartItemType[]>("cart") ?? [];
+
+  return (
+    <>
+      <div id="modal-overlay" css={ModalOverlay} onClick={onClickOverlay} />
+      <div css={ModalContainer}>
+        <h2 css={ModalTitle}>장바구니</h2>
+        <hr />
+        {cartItems?.map((data) => (
+          <CartItem {...data} />
+        ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)